File: examples/equos-nodejs-integration/src/list-voices.ts
import { client } from "./client.js";

const { total, voices } = await client.voices.listVoices({ take: 50 });

console.log(`${voices.length} of ${total} voice(s):`);
for (const v of voices) {
  console.log(`- ${v.id} · ${v.identity}`);
}

File: examples/equos-python-integration/src/python_integration/list_voices.py
from python_integration.client import get_client


def list_voices() -> None:
    client = get_client()
    response = client.voices.list(take=50)

    print(f"{len(response.voices)} of {response.total} voice(s):")
    for v in response.voices:
        print(f"- {v.id} · {v.identity.value}")


if __name__ == "__main__":
    list_voices()
